server/utils/validation/addUserValidation/index.js

```diff
@@ -0,0 +1,8 @@
+const { object, string } = require('yup');
+
+const addUserSchema = object({
+  email: string().email().required(),
+  password: string().min(8).required(),
+});
```

Comment:
I would prefer to add a confirmed password validation field, it's will be better for the first time sign up.

Reply:
that is the work of the front end it can be bypassed easily

Reply:
Actually, there's no front-end for this route currently and there might never be, so password confirmation at the validation schema wouldn't hurt anyone and it would spare our server a lot of unnecessary processes especially if there was a front-end and they bypassed it easily.